src/containers/CreateDoc/createDoc.css

```diff
@@ -0,0 +1,53 @@
+.page__content {
+  width: 100%;
+  max-width: 1024px;
+  margin: 0 auto;
+  display: grid;
+  grid-template: auto auto 1fr/ 30% 70%;
```

Comment:
NOT RELATED: Never used that before, seems usefull, but not sure if I understand it

Reply:
Its one of the multiple ways of creating a layout with the grid property.
You might know that while flexbox is used for one-dimensional layouts, the grid is very usefull when another axis comes to play (x-y)
Specifically, the grid-template is defining the composition of that template, separating the rows and the columns specs with a '/' (rows Specs / columns Specs)
Each specs used the same kind of values: rows and columns are managed the same way.
Auto: The row will took the height its content has
1fr   : Assume we want a 3 rows template and every row (or column) has 1fr as value (1fr 1fr 1r), every row would take `1/(1+1+1)` of the total height the tempalte end up occupying
%%: Each % value will tell that column to occupy that width in %

Summing up: in the example we have a grid template with 3 rows, where the first 2 will have the height of its content and two columns with the specific widths (30 % and 70 %)

Reply:
that sounds really complex, but usefull, I'll play aroudn with it, thanks for the explaination : )